bears/yaml/YAMLLintBear.py

```diff
@@ -33,13 +33,12 @@ def generate_config(filename, file,
         yamllint_configs = {
             'extends': 'default',
             'rules': {
-                'document-start': {
-                    'present': False
-                 }
+                'document-start': 'disable',
             }
         }
-        if document_start:
-            yamllint_configs['rules']['document-start']['present'] = True
+        if document_start is not None:
+            yamllint_configs['rules']['document-start'] = {
+                'present': document_start}
```

Comment:
For the sake of readability I think this is better:

```
    ...
    'rules': {}
    ...

# Don't forget to wrap the line to meet the 80 chars limit :D
yamllint_configs['rules']['document-start'] = 'disable' if document_start is None else {'present': document_start}
```

Instead of patching the dict depending on a value, we directly assign the correct value :)